spec/models/form/sales/questions/number_of_others_in_property_spec.rb

```diff
@@ -9,15 +9,15 @@
   let(:joint_purchase) { true }
 
   it "has correct page" do
-    expect(question.page).to eq(page)
+    expect(question.page).to be(page)
```

Comment:
parentheses not needed. Kat asked me to leave them in in general for consistency with the rest of the project. I sometimes don't type them if I only have one hand at the time.

In this case I think we do care about the object specifically as we want to be able to call question.page.bla but also in practice == equality for custom objects checks they are the same class and that they have the same id, so ultimately it will be the same test.
How do you think it adds copmlexity?

Reply:
In general my instinct is to be more specific where possible, so here be rather than eq